std/container/array.d

```diff
@@ -1585,6 +1585,21 @@ if (!is(Unqual!T == bool))
     ai.insertBack(arr);
 }
 
+// destructor std.container.array.Array!(MyClass).Array.~this is @system
+// so the unittest is @system too
```

Comment:
Sorry to have missed this the last time around, I think this just needs a reference (with a link) to the issue on bugzilla and this should be good to go.

Reply:
Sorry that I didn't do it myself. I promise the next time I'll be a good boy!